Protacon.RxMq.AzureServiceBus/Topic/AzureTopicPublisher.cs

```diff
@@ -86,14 +86,99 @@ public void Dispose()
                 .ToList()
                 .ForEach(x => x.Dispose());
         }
+
         public Task SendAsync<T>(T message) where T : new()
         {
             var topic = _settings.TopicNameBuilder(message.GetType());
 
             if (!_bindings.ContainsKey(topic))
-                _bindings.Add(topic, new Binding(_settings, _topicManagement, topic, typeof(T), _logger));
+                return TryCreateBinding(topic, typeof(T), message, 5, 60);
```

Comment:
Give magical numbers some names. I would also recommend using `TimeSpan` directly here instead of int for seconds.

Reply:
Added these to class fields